pkgs/tools/misc/cutecom/default.nix

```
@@ -1,25 +1,33 @@
-{ stdenv, fetchFromGitHub, qtbase, qtserialport, cmake }:
+{ mkDerivation, lib, fetchFromGitLab, qtbase, qtserialport, cmake, makeWrapper }:
 
-stdenv.mkDerivation rec {
+mkDerivation rec {
   pname = "cutecom";
-  version = "0.50.0";
-  src = fetchFromGitHub {
-    owner = "neundorf";
-    repo = "CuteCom";
+  version = "0.51.0";
+
+  src = fetchFromGitLab {
+    owner = "cutecom";
+    repo = "cutecom";
     rev = "v${version}";
-    sha256 = "0zjmbjrwwan9z5cphqjcq2h71cm4mw88j457lzdqb29cg4bdn3ag";
+    sha256 = "1zprabjs4z26hsb64fc3k790aiiqiz9d88j666xrzi4983m1bfv8";
   };
 
   preConfigure = ''
-    substituteInPlace CMakeLists.txt --replace "#find_package(Serialport REQUIRED)" "find_package(Qt5SerialPort REQUIRED)"
+    substituteInPlace CMakeLists.txt \
+      --replace "#find_package(Serialport REQUIRED)" "find_package(Qt5SerialPort REQUIRED)"
+  '';
+
+  buildInputs = [ qtbase qtserialport cmake makeWrapper ];
```

cmake should be a nativeBuildInputs, everything else should be taken care of by the libsForQt5.callPackage
```suggestion
  nativeBuildInputs = [ cmake ];
```

---

I'm not quite sure about your suggested diff. If I am also removing `qtserialport` the build will fail.

Please take a look at the updated PR. Thanks a lot and excuse my late reply.

---

So those dependencies are needed, the callPackage function just ensure they're from the correct scope. But `cmake` should be native :+1:

---

Thanks, I have updated the PR